### PR TITLE
Editorial nits

### PR DESCRIPTION
**Pull request by sthagen**
Just some small editorial enhancements:
* updated refs
* replaced http occurrences with https
* enhanced language in two or three places
* fixed typo
* updated date and changes table

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -1655,7 +1655,7 @@ If an issuing party decides to change its Namespace it SHOULD reissue all CSAF d
 Example:
 
 ```
-    http://www.example.com
+    https://www.example.com
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:1658]: Is it required to have `https` for the `namespace`? If so, should we state that explicitly? Or do we guide by example?

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 2]
```diff
@@ -329,9 +331,7 @@ _Package URL (PURL)_, GitHub Project, https://github.com/package-url/purl-spec
 ###### [RFC3552]
 Rescorla, E. and B. Korver, "Guidelines for Writing RFC Text on Security Considerations", BCP 72, RFC 3552, DOI 10.17487/RFC3552, July 2003, https://www.rfc-editor.org/info/rfc3552.
 ###### [RFC7231]
-Fielding, R. and J. Reschke, "Hypertext Transfer Protocol (HTTP/1.1): Semantics and Content", RFC 7231, DOI 10.17487/RFC7231, June 2014, https://www.rfc-editor.org/info/rfc7231.
-###### [RFC7464]
-N. Williams., "JavaScript Object Notation (JSON) Text Sequences", RFC 7464, DOI 10.17487/RFC7464, February 2015, http://www.rfc-editor.org/info/rfc7464.
+Fielding, R., Ed., and J. Reschke, Ed., "Hypertext Transfer Protocol (HTTP/1.1): Semantics and Content", RFC 7231, DOI 10.17487/RFC7231, June 2014, https://www.rfc-editor.org/info/rfc7231.
```
Comment (tschmidtb51) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:334]: Isn't the `Ed.`just a remark that they are editors?

**Review by tschmidtb51 — APPROVED** [review]
Just some general questions/comments but I totally approve these changes.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1658]: The latter is my intention. These non-authentic URLs are mostly accidental or historical ballast. I suggest, we do strongly emphasize authenticity and cryptographic hardness, so, we should maintain that level for many references that are extensions of the CSAF document (by reference).

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 2 above)
Comment (sthagen) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:334]: It is, but as stated in the commit message / PR comments, IETF RFC editor pages offer the citation format which I would like to simply inject. B.t.w. The editors remark is IMO an important part of the author information … if given.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:1658]: Agreed. Thanks for clarifying that.

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 2 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:334]: I see - thanks for the clarification.

**Pull request merged**